docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -1,301 +1,166 @@
 
-Note, from the REPL or scripts one can always use [`savegraph`](@ref) to save
-the graph (assuming `Graphviz` is installed).
+## [Simulating Catalyst models](@id introduction_to_catalyst_model_simulation)
+There exist three primary modes of simulation CRN models:
+1. Using ordinary differential equations (ODEs).
+2. Using stochastic differential equations (SDEs).
+3. Using jump process simulations.
 
-## Mass action ODE models
-Let's now use our `ReactionSystem` to generate and solve a corresponding mass
-action ODE model. We first convert the system to a `ModelingToolkit.ODESystem`
-by
-```@example tut1
-repressilator = complete(repressilator)
-odesys = convert(ODESystem, repressilator)
-```
-(Here Latexify is used automatically to display `odesys` in Latex within Markdown
-documents or notebook environments like Pluto.jl.)
+Catalyst models can be simulated using all three modes. Below we give a simple introduction to each, with [more detailed documentation provided elsewhere](@ref ref).
 
-Before we can solve the ODEs, we need to specify the values of the parameters in
-the model, the initial condition, and the time interval to solve the model on.
-To do this we need to build mappings from the symbolic parameters and the
-species to the corresponding numerical values for parameters and initial
-conditions. We can build such mappings in several ways. One is to use Julia
-`Symbols` to specify the values like
-```@example tut1
-pmap  = (:α => .5, :K => 40, :n => 2, :δ => log(2)/120,
-         :γ => 5e-3, :β => log(2)/6, :μ => log(2)/60)
-u₀map = [:m₁ => 0., :m₂ => 0., :m₃ => 0., :P₁ => 20., :P₂ => 0., :P₃ => 0.]
-nothing   # hide
+### [ODE simulations](@id introduction_to_catalyst_model_simulation_ode)
+Let us consider the infectious disease model declared in the previous section. To simulate it we need (in addition to the model):
+* The simulation's initial condition. That is, the concentration (or copy numbers) of each species at the start of the simulation.
+* The simulation's timespan. That is, the time frame over which we wish to run the simulation.
+* The simulation's parameter values. That is, the values of the model's parameters for this simulation.
+
+The initial conditions and parameter values are declared as vectors of pairs. Here, each entry pairs the symbol corresponding to a specific quantity's name to its value (e.g. `:S => 99` denotes that the initial concentration of `S` should be `99`). The timespan is simply a tuple with the simulations *starting* and *final* times. Once we have set these values, we collect them in a so-called `ODEProblem`.
+```@example intro_1
+u0 = [:S => 99, :I => 1, :R => 0]
+tspan = (0.0, 20.0)
+ps = [:β => 0.025, :γ => 0.2]
+oprob = ODEProblem(sir_model, u0, tspan, ps)
 ```
-Alternatively, we can use ModelingToolkit-based symbolic species variables to
-specify these mappings like
-```@example tut1
-t = default_t()
```

Comment:
All MTK and internal stuff is moved to the thrid intro page (Advanced introduction to Catalyst, or something like that, not sure what we think the best name is)